javascriptv3/example_code/s3/actions/delete-bucket-website.js

```diff
@@ -1,29 +1,61 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-import { fileURLToPath } from "url";
-
 // snippet-start:[s3.JavaScript.website.deleteBucketWebsiteV3]
-import { DeleteBucketWebsiteCommand, S3Client } from "@aws-sdk/client-s3";
-
-const client = new S3Client({});
+import {
+  DeleteBucketWebsiteCommand,
+  S3Client,
+  S3ServiceException,
+} from "@aws-sdk/client-s3";
 
-// Disable static website hosting on the bucket.
-export const main = async () => {
-  const command = new DeleteBucketWebsiteCommand({
-    Bucket: "test-bucket",
-  });
+/**
+ * Remove the website configuration for a bucket.
+ * @param {{ bucketName: string }}
+ */
+export const main = async ({ bucketName }) => {
+  const client = new S3Client({});
 
   try {
-    const response = await client.send(command);
-    console.log(response);
-  } catch (err) {
-    console.error(err);
+    await client.send(
+      new DeleteBucketWebsiteCommand({
+        Bucket: bucketName,
```

Comment:
If this also deletes the bucket, is there a check to make sure it's empty beforehand? E.g handling of `BUcketNotEmpty` error.

Reply:
This does not delete a bucket. It disables the bucket's configuration as a website.